tests/FSharp.Compiler.ComponentTests/Conformance/BasicGrammarElements/CustomAttributes/CallerArgumentExpression/CallerArgumentExpression.fs

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Conformance.BasicGrammarElements
+
+open Xunit
+open FSharp.Test.Compiler
+open FSharp.Test
+
+module CustomAttributes_CallerArgumentExpression =
```

Comment:
Did you consider a de-generate scenario of a huge file being a single expression (e.g. a massive collection builder, or a massive async block) passed into a single `CallerArgumentExpressionAttribute` member?